tests/flow/test_generic_cmds.py

```diff
@@ -135,6 +135,19 @@ def test_restore():
                                               [b't{1}_agg', [], []]
                                               ]))
 
+<<<<<<< HEAD
```

Comment:
@OfirMos you need to fix the conflict

Reply:
I don't understand this git, why it's a conflict 😠